### PR TITLE
Add contributing.md first draft

### PR DESCRIPTION
**Pull request by tatlax3636**
Initial draft of contribution guidelines. Please add any comments/suggestions!

### CODE REVIEW

File: CONTRIBUTING.md
Comment (ConnorCason) [inline CONTRIBUTING.md:3]: `It is because of developers like you that this project is able to succeed.` --> `Developers like you are what allows this project to succeed`

Wording is slightly awkward, but not sure my suggestion is better.

File: CONTRIBUTING.md
Comment (ConnorCason) [inline CONTRIBUTING.md:5]: `that should be followed` --> `to follow`

File: CONTRIBUTING.md
Comment (ConnorCason) [inline CONTRIBUTING.md:9]: `update multiple things` --> `add or improve more than one functionality`

`please seperate into` --> `please separate your changes into`

File: CONTRIBUTING.md
Comment (ConnorCason) [inline CONTRIBUTING.md]: Possibly collapse this since every benchmark is 80%. Maybe something like:
`Hygieia requires wholistic code coverage above the 80% mark, including Branch, Line, Statement, and Function coverage`

Once again, simple preference.

**Review by rvema — APPROVED** [review]
lgtm

File: CONTRIBUTING.md
Comment (ConnorCason) [inline CONTRIBUTING.md:17]: `has at least 80% coverage` --> `abides by the guidelines provided`

File: CONTRIBUTING.md
Comment (ConnorCason) [inline CONTRIBUTING.md:22]: I want to say the email is `hygieia2@capitalone.com` but double check that before merging.

**Review by ConnorCason — COMMENTED** [review]
Great job writing this up, my suggestions are all small wording changes that may help the document read slightly better. Up to you which ones you want to revise! Happy to approve when you're done with final edits.

File: CONTRIBUTING.md
Comment (tatlax3636) [reply CONTRIBUTING.md:22]: @rvema is this the right email? I don't want to put C1 specific info without checking

**Review by ConnorCason — APPROVED** [review]
LGTM

**Review by danielyhuang — APPROVED** [review]
Lgtm

**Pull request merged**
